pages/linux/lastcomm.md

```diff
@@ -0,0 +1,20 @@
+# lastcomm
+
+> Show last commands executed (from acct - GNU Accounting utilities).
```

Comment:
Is the part "(from acct - GNU Accounting utilities)" really necessary? 

The idea of tldr is to keep things simple, providing a simple description with practical examples. In my opinion, this information is not necessary for a tldr page.

Reply:
removed.